versions/tasks.py

```diff
@@ -405,6 +415,25 @@ def get_release_date_for_version(version_pk, commit_sha, token=None):
         logger.error("get_release_date_for_version_error", version_pk=version_pk)
 
 
+@app.task
+def purge_fastly_release_cache():
+    if not settings.FASTLY_API_TOKEN or settings.FASTLY_API_TOKEN == "empty":
```

Comment:
Should it `return True` or something similar here?   
After detecting that `settings.FASTLY_API_TOKEN == "empty"` and logging "Not purging cache", is it exiting?

Reply:
or wait....    If an `else` statement is added, that's another option also.  Maybe better. Right?

Reply:
I definitely meant for there to be a return!  Good catch.  I'm partial to the guard clause and early breaking with a return, personally.